doc/explanations/exercises.rst

```diff
@@ -0,0 +1,104 @@
+=======================
+Native tokens exercises
```

Comment:
This documentation should probably live in the cardano-node repository.

Reply:
Noted, thanks! I'll delete it from here and talk to the node team to host this in 'references'